department-of-veterans-affairs/abd-vro
# MCP-2211 Add 400 Bad Request responses to swagger list for MAS

## What was the problem?
400 responses are missing in the list of responses for MAS endpoints on swagger

Associated tickets or Slack threads:

- [MCP-2211](https://amida.atlassian.net/browse/MCP-2211)

## How does this fix it?[^1]
Adds 400 "Bad Request" to response list in swagger.

## How to test this PR
- Step 1 run the app
- Step 2 verify that 400 bad request is in the list of responses for MAS endpoints.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
